android/gradle.properties

```diff
@@ -21,4 +21,4 @@ org.gradle.jvmargs=-Xmx2048m --add-exports jdk.compiler/com.sun.tools.javac.api=
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
-djl_version=0.19.0-SNAPSHOT
+djl_version=0.20.0-SNAPSHOT
```

Comment:
we'd like to use release version in djl-demo if possible. use -SNAPSHOT only we add new demo that requires -SNAPSHOT api

Reply:
OK